js/containers/portal-dashboard/feedback/feedback-info.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { connect } from "react-redux";
+import {FeedbackNoteToggle} from "../../../components/portal-dashboard/feedback/feedback-note-toggle";
+import {FeedbackLegend} from "../../../components/portal-dashboard/feedback/feedback-legend";
+import { FeedbackLevel, ListViewMode } from "../../../util/misc";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-info.less";
+
+interface IProps {
+  feedbackLevel: FeedbackLevel;
+  listViewMode: ListViewMode;
+  rubric?: any;
```

Comment:
Does rubric have a type?  Probably not worth worrying about in this PR.  We can sort it out with the rubric work.

Reply:
I don't see one anywhere.